src/main.rs

```diff
@@ -113,7 +114,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     // Load the config.
     let mut config_created = false;
-    let path = cli.config.or_else(|| {
+    let path = cli.config.or_else(env_config_path).or_else(|| {
```

Comment:
`NIRI_CONFIG` should be removed from the environment somewhere below the last use.